app/src/main/java/jp/co/cyberagent/stf/Service.java

```diff
@@ -367,4 +373,69 @@ public void run() {
             }
         }
     }
+
+    /**
+     * Monitors the adb state by checking /sys/class/android_usb/android0/state
+     *
+     * If state is DISCONNECTED then IdentityActivity will be shown to allow easier identification
+     * of malfunctioning devices
+     */
+    private class AdbMonitor extends Thread {
+        // If something goes wrong you have 30 seconds to kill the STFService
+        // Using smaller numbers will basically lock the devices until usb connection is established
+        private static final int INTERVAL_MS = 5000;
+
+        @Override
+        public void run() {
+            Log.d(TAG, "Starting adb monitor thread");
+            java.lang.Process process;
+            try {
+                while(!isInterrupted()) {
```

Comment:
Hey can you try to make sure there's a space before every `while` and `if`? That way it matches with the current format.

Reply:
Ofc